tests/Common/Common.h

```diff
@@ -1227,3 +1227,7 @@ enum EmptyEnum { };
 enum __enum_with_underscores { lOWER_BEFORE_CAPITAL, CAPITALS_More, underscore_at_end_, usesDigits1_0 };
 
 void DLL_API sMallFollowedByCapital();
+
```

Comment:
Add a function which uses this struct and call that function in the test code. We must ensure such functions are generated. In fact, you don't need to use a new incomplete structure. The one we already have - I believe it's called "IncompleteStruct" - should do just fine, simply forward it in AnotherUnit.h as well.